core/backend/src/IModelDb.ts

```diff
@@ -1147,6 +1147,20 @@ export abstract class IModelDb extends IModel {
   public exportPartGraphics(exportProps: ExportPartGraphicsOptions): DbResult {
     return this.nativeDb.exportPartGraphics(exportProps);
   }
+
+  /** Request geometry stream information from an element.
+   * @alpha
+   */
+  public elementGeometryRequest(requestProps: ElementGeometryRequest): DbResult {

```

Comment:
nitpick but these names are awkward. Wouldn't `updateElementGeometry` and `request/processElementGeometry` make more sense?